docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -58,7 +58,14 @@ You can analyze more than one application at a time against more than one transf
 * Exclude packages. If you choose this option, type the name of the package name and click *Add*.
 
 . Click *Next*.
-. In *Advanced*, set *Custom rules* by typing a name or by searching and clicking *Add Rules*.
+. In *Advanced*, you can attach additional custom rule to the analysis. Select *Manual* or *Repository*.
```

Comment:
Grammar error - **you can attach additional custom rule**
This must either be a plural  or there should be include a determiner before **additional** i.e. **a**